pallets/pools/src/lib.rs

```diff
@@ -789,7 +798,12 @@ pub mod pallet {
 		pub fn update(
 			origin: OriginFor<T>,
 			pool_id: T::PoolId,
-			changes: PoolChanges<T::InterestRate, T::MaxTokenNameLength, T::MaxTokenSymbolLength>,
+			changes: PoolChanges<
```

Comment:
I think same here, `PoolChangesOf<T>` would be nice to have

Reply:
Done